foxpuppet/windows/browser.py

```diff
@@ -2,43 +2,56 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from foxpuppet.windows import Windows
-
 from selenium.webdriver.common.by import By
 
+from ..expected import new_browser_window_is_opened
+from ..windows.base import BaseWindow
```

Comment:
I think we should use absolute imports. I mentioned we could do this in a later commit, but I'm not keen on this change going in.